src/Components/HistoryData.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Fade } from 'react-reveal'
+
+export default function HistoryData({ ID, name, explanation, month, year }) {
+    return (
+        <Fade delay={300*ID}>
+            <div style={{ marginTop: 300 * ID }} className="circular-img">
+
+                {
+                    ID % 2 == 0 ? <div className="caption">
```

Comment:
this logic is used twice, we can have a flag for it and use it; `const isEven = id%2==0`